robot-core/src/main/java/org/obolibrary/robot/ExportOperation.java

```diff
@@ -0,0 +1,739 @@
+   *
+   * @param classes Set of class expressions to convert to string
+   * @param provider ShortFormProvider for Manchester expressions
+   * @param excludeAnonymous if true, exclude anonymous class expressions
+   * @return String of class expressions or null
+   */
+  private static String classExpressionsToString(
+      Collection<OWLClassExpression> classes,
+      ShortFormProvider provider,
+      boolean excludeAnonymous) {
+    List<String> strings = new ArrayList<>();
+    for (OWLClassExpression expr : classes) {
+      if (expr.isAnonymous() && !excludeAnonymous) {
+        // Get a Manchester string using labels
+        String manString = renderManchester(expr, provider);
+        strings.add(manString);
+      } else if (!expr.isAnonymous()) {
+        OWLClass sc = expr.asOWLClass();
+        strings.add(provider.getShortForm(sc));
+      }
+    }
+    if (strings.isEmpty()) {
+      return null;
+    }
+    List<String> sorted =
+        strings.stream().sorted(String::compareToIgnoreCase).collect(Collectors.toList());
+    return String.join("|", sorted);
+  }
+
+  /**
+   * Given an ontology, a quoted entity checker, an entity, a list of columns, a label function, and
+   * a boolean indicating to exclude anonymous classes, return a map of cell values for this entity.
+   *
+   * @param ontology OWLOntology to get details from
+   * @param checker QuotedEntityChecker to resolve entities
+   * @param e OWLEntity to get details of
+   * @param columns List of columns indicating the details to get
+   * @param provider ShortFormProvider for Manchester expressions
+   * @param excludeAnonymous if true, exclude anonymous class expressions
+   * @return Map of column names to cell values
+   * @throws Exception if a property (from columns) cannot be resolved
+   */
+  private static Map<String, String> getCellValues(
+      OWLOntology ontology,
+      QuotedEntityChecker checker,
+      OWLEntity e,
+      List<String> columns,
+      ShortFormProvider provider,
+      boolean excludeAnonymous)
+      throws Exception {
+    Map<String, String> cellMap = new HashMap<>();
+    for (String col : columns) {
+      switch (col) {
+        case "IRI":
+          cellMap.put("IRI", e.getIRI().toString());
+          break;
+        case "CURIE":
+          cellMap.put("CURIE", e.getIRI().getShortForm().replace("_", ":"));
+          break;
+        case "LABEL":
+          cellMap.put("LABEL", provider.getShortForm(e));
+          break;
+        case "subclass-of":
+        case "rdfs:subClassOf":
+          if (e.isOWLClass()) {
+            cellMap.put(
+                col,
+                classExpressionsToString(
+                    EntitySearcher.getSuperClasses(e.asOWLClass(), ontology),
+                    provider,
+                    excludeAnonymous));
+          }
+          break;
+        case "subproperty-of":
+        case "rdfs:subPropertyOf":
```

Comment:
These case statements should match IRIs (previously resolved) not CURIEs. Even though the conventions are strong for prefixes such as `rdfs`, people are allowed to use whatever prefixes they want.

Reply:
Resolved to use CURIE to get full IRI